plutus-core/cost-model/budgeting-bench/Generators.hs

```diff
@@ -10,7 +11,9 @@ import Control.Monad
 import Data.Bits
 import Data.ByteString (ByteString)
 import Data.Int (Int64)
+#if !MIN_VERSION_base(4,20,0)
 import Data.List (foldl')
+#endif
```

Comment:
Any way we could do this without the CPP?

Reply:
I removed it somewhere else so will do the same here and the other places.

Reply:
Fixed.